app/bundles/CoreBundle/Assets/json/countries.json

```diff
@@ -99,50 +119,68 @@
     "Liechtenstein",
     "Lithuania",
     "Luxembourg",
+    "Macao",
+    "Macedonia",
     "Madagascar",
     "Malawi",
     "Malaysia",
     "Maldives",
     "Mali",
     "Malta",
     "Marshall Islands",
+    "Martinique",
     "Mauritania",
     "Mauritius",
+    "Mayotte",
     "Mexico",
     "Micronesia",
     "Moldova",
     "Monaco",
     "Mongolia",
     "Montenegro",
+    "Montserrat",
     "Morocco",
     "Mozambique",
     "Myanmar",
     "Namibia",
     "Nauru",
     "Nepal",
     "Netherlands",
+    "New Caledonia",
     "New Zealand",
     "Nicaragua",
     "Niger",
     "Nigeria",
+    "Niue",
     "North Korea",
```

Comment:
Ah, North Korea is here, but that's not it's correct name (see note above in the 'K' section on the two Koreas)